### PR TITLE
Stories improvements

### PR DESCRIPTION
**Pull request by tomirisk**
Stories created during last 24h are shown

### CODE REVIEW

File: controllers/story-controller.ts [hunk 1]
```diff
@@ -137,8 +137,16 @@ export default class StoryController implements StoryControllerI {
    * @param {Response} res Represents response to client, including the
    * body formatted as JSON arrays containing the story objects
    */
-  findStories = (req: Request, res: Response) => {
-      StoryController.storyDao.findStories().then((stories: Story[]) => res.json(stories));
+  findStories = async (req: Request, res: Response) => {
+    const stories = await StoryController.storyDao.findStories();
+    const twentyFourHours = new Date().getTime() + (24 * 60 * 60 * 1000);
```
Comment (sghosalkar) [inline controllers/story-controller.ts]: We should be accepting hours as a parameter
`
if (req.query.hours) {
    const hours = new Date().getTime() - (req.query.hours * 60 * 60 * 1000);
    const filteredStories = stories.filter((story) => hours <= story.postedOn.getTime());
      return res.json(filteredStories);
    }
`

Comment (tomirisk) [reply controllers/story-controller.ts]: Fixed!

File: controllers/story-controller.ts [hunk 2]
```diff
@@ -137,8 +137,16 @@ export default class StoryController implements StoryControllerI {
    * @param {Response} res Represents response to client, including the
    * body formatted as JSON arrays containing the story objects
    */
-  findStories = (req: Request, res: Response) => {
-      StoryController.storyDao.findStories().then((stories: Story[]) => res.json(stories));
+  findStories = async (req: Request, res: Response) => {
+    const stories = await StoryController.storyDao.findStories();
+    if (req.query.hours) {
+      const twentyFourHoursBefore = Date.now() - (parseInt(String(req.query.hours)) * 60 * 60 * 1000);
```
Comment (sghosalkar) [inline controllers/story-controller.ts]: Could be renamed to `hoursOffset` or `hoursBefore`?

File: controllers/story-controller.ts [hunk 3]
```diff
@@ -148,15 +156,24 @@ export default class StoryController implements StoryControllerI {
...
     if(userUid === "me"){
       res.sendStatus(503);
       return;
     }
     try {
-      StoryController.storyDao.findStoriesVisibleToUser(userUid).then((stories: Story[]) => res.json(stories));
+      const stories = await StoryController.storyDao.findStoriesVisibleToUser(userUid);
+
+      if (req.query.hours) {
+        const twentyFourHoursBefore = Date.now() - (parseInt(String(req.query.hours)) * 60 * 60 * 1000);
```
Comment (sghosalkar) [inline controllers/story-controller.ts]: `parseInt(req.query.hours as string)`

**Review by sghosalkar — APPROVED** [review]

**Review by galen94 — APPROVED** [review]

**Pull request merged**